src/components/Modal/CommentOptionModal.js

```diff
@@ -7,7 +7,7 @@ import {COLORS} from '../../utils/theme';
 
 const styles = StyleSheet.create({
   container: {
-    backgroundColor: COLORS.white,
+    backgroundColor: COLORS.almostBlack,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `COLORS.white` to `COLORS.almostBlack` for the background color of the container might affect the readability and contrast of any text or components within this container. Please ensure that all child components of this container have been checked and updated if necessary to maintain good contrast and readability.
```diff
-    backgroundColor: COLORS.white,
+    backgroundColor: COLORS.almostBlack,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->